exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,11 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  if(sentence.includes("code")) {
```

Comment:
you could just do `return sentence.includes("code")` to simplify your function :)